internal/nodes/dequeue_ingestion.go

```diff
@@ -743,7 +743,7 @@ func getIngressNSNameForIngestion(objType, namespace, nsname string) (string, st
 		return arr[0], arr[1]
 	}
 
-	if objType == utils.IngressClass || objType == lib.AviInfraSetting {
+	if objType == utils.IngressClass || objType == lib.AviInfraSetting || objType == utils.NamespaceNetworkInfo {
```

Comment:
@abhishekbsingh: As per our discussion, should we revert this change?

Reply:
done